Client/src/Components/related-products/ProductCard.jsx

```diff
@@ -1,21 +1,21 @@
-/* eslint-disable import/extensions */
 import React, { useState, useEffect } from "react";
 import PropTypes from "prop-types";
 import axios from "axios";
 import css from "./styles.css";
 import helper from "./helper-functions";
+import StarList from "../product-reviews/star-rating/StarList";
 
 const { getTruePrice } = helper;
```

Comment:
You could actually move this up to line 5 as 'import { getTruePrice } from "./helper-functions";'.

Reply:
Webpack will throw an error if I do it all on the same line.

Reply:
Huh, that's really odd, I was able to destructure right out of the import in a few places... boo webpack